lib/ansible/modules/cloud/linode/linode.py

```diff
@@ -597,18 +591,19 @@ def main():
                                                      'restarted']),
             api_key = dict(no_log=True),
             name = dict(type='str'),
-            alert_bwin_enabled = dict(type='bool', default=True),
-            alert_bwin_threshold = dict(type='int'),
-            alert_bwout_enabled = dict(type='bool', default=True),
-            alert_bwout_threshold = dict(type='int'),
-            alert_bwquota_enabled = dict(type='bool', default=True),
-            alert_bwquota_threshold = dict(type='int'),
-            alert_cpu_enabled = dict(type='bool', default=True),
-            alert_cpu_threshold = dict(type='int'),
-            alert_diskio_enabled = dict(type='bool', default=True),
-            alert_diskio_threshold = dict(type='int'),
-            backupweeklyday = dict(type='int'),
-            backupwindow = dict(type='int'),
+            alert_bwin_enabled = dict(type='bool', default=None),
+            alert_bwin_threshold = dict(type='int', default=None),
+            alert_bwout_enabled = dict(type='bool', default=None),
+            alert_bwout_threshold = dict(type='int', default=None),
+            alert_bwquota_enabled = dict(type='bool', default=None),
+            alert_bwquota_threshold = dict(type='int', default=None),
+            alert_cpu_enabled = dict(type='bool', default=None),
+            alert_cpu_threshold = dict(type='int', default=None),
+            alert_diskio_enabled = dict(type='bool', default=None),
```

Comment:
The `default` here previously was `True` now `None`.  The docs state that this is boolean, and the default is `True`.

I assume setting it to `None` changes the behavior in a non-backwards compatible way.